src/SFA.DAS.EAS.Portal.Client/IPortalClient.cs

```diff
@@ -10,10 +10,10 @@ public interface IPortalClient
         /// Get the account details required to render the homepage.
         /// </summary>
         /// <param name="hashedAccountId">The (non-public) hashed account id.</param>
-        /// <param name="accountState">The current state of the account. Set all flags that apply.</param>
+        /// <param name="maxNumberOfVacancies">The maximum number of vacancies to return.</param>
         /// <param name="cancellationToken">An optional cancellation token.</param>
         /// <returns></returns>
-        Task<Account> GetAccount(string hashedAccountId, AccountState accountState,
+        Task<Account> GetAccount(string hashedAccountId, int maxNumberOfVacancies,
```

Comment:
Again I think this should be a filter class of some sort with a int MaxNumberOfVacancies property to make this extensible without changing the signature of the method all the time.

Reply:
A parameter object has now been introduced and the interface is agreed - unblocking the front end stories.